src/realm-settings/TokensTab.tsx

```diff
@@ -402,6 +403,32 @@ export const RealmSettingsTokensTab = ({
               )}
             />
           </FormGroup>
+          <FormGroup
+            label={t("defaultAudValue")}
+            fieldId="defaultAudValue"
+            id="default-aud-value-label"
+            labelIcon={
+              <HelpItem
+                helpText="realm-settings-help:defaultAudValueTooltip"
+                fieldLabelId="realm-settings:defaultAudValue"
+              />
+            }
+          >
+            <Controller
+              name="defaultAudValueForAccessToken"
+              defaultValue={""}
```

Comment:
Braces can be omitted if the value is a string literal.
```suggestion
              defaultValue=""
```